event_routing_backends/models.py

```diff
@@ -240,6 +245,33 @@ def _match_event_for_host(self, original_event, host_config):
             bool
         """
         for key, value in host_config.get('match_params', {}).items():
-            if get_value_from_dotted_path(original_event, key) != value:
+            original_event_value = get_value_from_dotted_path(original_event, key)
+            if isinstance(value, list):
+                matched = False
+                for value_item in value:
+                    if self._is_match(value_item, original_event_value):
+                        matched = True
+                if not matched:
+                    return False
+            elif not self._is_match(value, original_event_value):
                 return False
         return True
```

Comment:
by default `_match_event_for_host` should return False

Reply:
As we are consider an AND relationship between different attributes of match params. But this logic will make an OR relationship between attributes